### PR TITLE
chore: cleanup buttons

### PR DESCRIPTION
**Pull request by simboonlong**

### CODE REVIEW

File: src/components/Container/Container.tsx [hunk 1]
```diff
@@ -2,8 +2,8 @@ import React, { FunctionComponent } from "react";
 
 export const Container: FunctionComponent = ({ children }) => {
   return (
-    <div className="bg-blue-300 min-h-screen">
-      <div className="mx-auto">{children}</div>
-    </div>
+    <main className="bg-blue-300">
+      <div className="container">{children}</div>
+    </main>
```
Comment (isaackps) [inline src/components/Container/Container.tsx:7]: how come change to <main>? im afraid some component is wrapped in a <article>, <aside>, <footer>, <header>, or <nav> then the main got problem liao.

Comment (simboonlong) [reply src/components/Container/Container.tsx:7]: right... let me look into this more

File: src/components/AddressResolverContainer/AddressResolverContainer.tsx [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+import { Wrapper } from "../UI/Wrapper";
 
 export const AddressResolverContainer: FunctionComponent = () => {
   return (
     <>
       <NavigationBar />
-      <AddressResolver />
+      <Wrapper>
+        <AddressResolver />
+      </Wrapper>
```
Comment (isaackps) [inline src/components/AddressResolverContainer/AddressResolverContainer.tsx:12]: I think this one dont need wrapper as the addressResolver from the ui component repo has the contianer classname liao.

Comment (simboonlong) [reply src/components/AddressResolverContainer/AddressResolverContainer.tsx:12]: oh ok thanks

**Review by isaackps — APPROVED** [review]

**Pull request merged**